tests/system/test/AssetsBridge.test.ts

```diff
@@ -0,0 +1,961 @@
+      gasCost = gasCost + receipt.gasUsed * receipt.gasPrice;
+    });
+
+    it("should verify the transaction did revert", async function() {
+      expect(receipt!.status).to.equal(0);
+    });
+
+    it("should verify the new balances", async function() {
+      var updatedSenderBalance = await ethers.provider.getBalance(senderAddress);
+      expect(updatedSenderBalance).to.equal(initialSenderBalance - gasCost - tokenAmount);
+    });
+
+    it("should verify that BTC and BTC ERC20 balance are equal", async function() {
+      expect(await ethers.provider.getBalance(senderAddress))
+        .to.equal(await btcToken.balanceOf(senderAddress));
+    });
+
+    it("should verify that contract address balances haven't changed", async function() {
+      expect(await ethers.provider.getBalance(contractAddress))
+        .to.equal(await btcToken.balanceOf(contractAddress))
+        .to.equal(ethers.parseEther("8"));
+    });
+  });
+
+  describe("contractBridgeOutBtcToBitcoinSuccess", function() {
+    let receipt: any;
+    let tx: any;
+    let tokenAmount: any;
+    let initialSenderBalance: any;
+    let recipient = "1976a91462e907b15cbf27d5425399ebf6f0fb50ebb88f1888ac";
+    let gasCost = 0;
+
+    before(async function() {
+      await fixture();
+      tokenAmount = ethers.parseEther("8");
+
+      initialSenderBalance = await ethers.provider.getBalance(senderAddress);
+
+      // send the funds to the contract first
+      const transferTx = await btcToken.connect(senderSigner).transfer(contractAddress, tokenAmount);
+      await transferTx.wait();
+      receipt = await ethers.provider.getTransactionReceipt(transferTx.hash);
+      gasCost = receipt.gasUsed * receipt.gasPrice;
+
+      tx = await bridgeOut.connect(senderSigner).bridgeOutBTCToBitcoinSuccess(
+        Buffer.from(recipient, "hex"),
+        tokenAmount,
+      );
+      await tx.wait();
+
+      receipt = await ethers.provider.getTransactionReceipt(tx.hash);
+      gasCost = gasCost + receipt.gasUsed * receipt.gasPrice;
+    });
+
+    it("should verify the transaction didn't revert", async function() {
+      expect(receipt!.status).to.equal(1);
+    });
+
+    it("should verify the new balances", async function() {
+      var updatedSenderBalance = await ethers.provider.getBalance(senderAddress);
+      expect(updatedSenderBalance).to.equal(initialSenderBalance - gasCost - tokenAmount);
+    });
+
+    it("should verify that BTC and BTC ERC20 balance are equal", async function() {
+      expect(await ethers.provider.getBalance(senderAddress))
+        .to.equal(await btcToken.balanceOf(senderAddress));
+    });
+
+    it("should verify that contract address have 0 balance", async function() {
+      expect(await ethers.provider.getBalance(contractAddress))
+        .to.equal(await btcToken.balanceOf(contractAddress))
+        .to.equal(0n);
+    });
+  });
+
+  describe("bridgeOutBTCToBitcoinSuccess", function() {
```

Comment:
We already have `bridgeOutBtcSuccess`. This case is redundant.

Reply:
02e29e42